WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -9,6 +9,7 @@ import CoreData
 protocol CommentDetailsNotificationNavigationDelegate: AnyObject {
```

Comment:
I just realized this protocol name isn't really accurate with the new method I added. I'll make a note to rename it in a future PR.